timbot.py

```diff
@@ -7,7 +7,7 @@
 
 slack_token = os.environ['SLACK_API_TOKEN']
 channel_name = os.environ['SLACK_CHANNEL_NAME']
-timbot_user_id = os.environ['SLACK_TIMBOT_USER_ID']
+timbot_user_id = os.environ['SLACK_TIMBOT_USER_ID'].lower()
```

Comment:
Why are we lowering the user id?

Reply:
because i lowered the message and the message contains the id. so if left capitalized it will not be matched during the check

Reply:
bet